index.js

```diff
@@ -11,7 +11,7 @@ const filterUnresolvedDeps = async (config, dependencies, devDependencies) => {
     email: config.email,
   });
 
-  const jiraTasks = await jira.searchDepcheckIssues({comment: config.comment, summary: `${github.context.repo.repo} ${SUMMARY}`});
+  const jiraTasks = await jira.searchDepcheckIssues({comment: config.comment, summary: `${github.context.repo.repo} - ${SUMMARY} ${config.comment}`});
```

Comment:
I don't remember why but jira search doesnt' work well with `-`

![image](https://github.com/optimaxdev/action-dependency_check/assets/10272661/a56ddac6-2c95-4527-baf6-5b0ae69d5851)
![image](https://github.com/optimaxdev/action-dependency_check/assets/10272661/ce328f4e-fcb8-4800-b7a5-e855a3ca3197)

Reply:
but without spaces it works) weird...
![image](https://github.com/optimaxdev/action-dependency_check/assets/10272661/79662885-1b69-4b9b-b9d2-d9d171e6f23e)

Reply:
https://jira.atlassian.com/browse/JRACLOUD-72895
there is the same problem
![image](https://github.com/optimaxdev/action-dependency_check/assets/10272661/748020dc-ecf6-4f97-b307-a9f44e4cc2f2)
so you can you use it with double slash

Reply:
done, thank you!